spec/features/features_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'takeaway'
+require 'menu'
+require 'twilio-ruby'
+
+  describe 'Feature Tests' do
+    subject (:takeaway) { Takeaway.new(Menu.new, Twilio::REST::Client.new("0", "0")) }
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.